acceptance/gossip_peerings_test.go

```diff
@@ -83,4 +84,20 @@ func TestGossipPeerings(t *testing.T) {
 	defer l.Stop()
 
 	checkGossipPeerings(t, l, 20*time.Second)
+
+	// Restart the first node.
+	log.Infof("Restarting node 0")
```

Comment:
Other log messages in the acceptance tests are not capitalized. Let's be consistent.